packages/design-system/src/components/web-components/ds-text-field/ds-text-field.stories.tsx

```diff
@@ -0,0 +1,119 @@
+import WebComponentDocTemplate from '../../../../../../.storybook/docs/WebComponentDocTemplate.mdx';
+import { action } from '@storybook/addon-actions';
+import { useEffect } from 'react';
+import { webComponentDecorator } from '../storybook';
+import './ds-text-field';
+
+export default {
+  title: 'Web Components/ds-text-field',
+  argTypes: {
+    'class-name': {
+      description: 'Additional classes to be added to the root element.',
+      control: 'text',
+    },
+    disabled: { description: 'Disables the entire field.', control: 'boolean' },
+    'error-message': {
+      description: 'Enable the error state by providing an error message',
+      control: 'text',
+    },
+    'error-id': {
+      description:
+        'The ID of the error message applied to this field. If none is provided, the id will be derived from the `root-id` attribute.',
+      control: 'text',
+    },
+    'error-placement': {
+      description: 'Location of the error message relative to the field input',
+      options: [undefined, 'top', 'bottom'],
+      control: { type: 'radio' },
+    },
+    'field-class-name': {
+      description: 'Additional classes to be added to the input element',
+      control: 'text',
+    },
+    hint: {
+      description: 'Hint text or HTML',
+      control: 'text',
+    },
+    'hint-id': {
+      description: 'The ID of the hint element',
+      control: 'text',
+    },
+    inversed: {
+      description: 'Set to "true" to apply the "inverse" theme',
+      control: 'boolean',
+    },
+    label: {
+      description: 'Label text or HTML.',
+      control: 'text',
+    },
+    'label-class-name': {
+      description: 'Additional classes to be added to the field label',
+      control: 'text',
+    },
+    'label-id': {
+      description:
+        "A unique `id` to be used on the field label. If one isn't provided, a unique ID will be generated.",
+      control: 'text',
+    },
+    'label-mask': {
+      description:
+        'Applies date format masking to the input value entered and renders to a text field above the input. Passing `true` to `valueOnly` will return just the formatted value entered.',
```

Comment:
What is `valueOnly` referring to in this description?

Reply:
Fixed in [7471e35](https://github.com/CMSgov/design-system/pull/3172/commits/7471e35d92614e90ca03b258d5d140dd21b096d7)